node/src/accountant/mod.rs

```diff
@@ -4933,6 +5042,65 @@ mod tests {
         );
     }
 
+    #[test]
+    fn failed_txs_were_there_so_payable_scan_is_rescheduled_as_retry_payable_scan_was_omitted() {
```

Comment:
Unclear name. The whole test does not mention any failed txs so do not refer to them in the name. 

`retry_payable_scan_is_requested_to_be_repeated`